css/input.css

```diff
@@ -0,0 +1,22 @@
+input:invalid:required {
+    border: 1px solid red;
+}
+input:invalid {
+    box-shadow: 0 0 1px 1px red;
+}
+input:valid:required {
+    background: url(../images/check.png);
+    background-size: 1rem;
+    background-repeat: no-repeat;
+    background-position: .5rem 0.4rem;
+}
+input {
+    padding-left: 1.8rem;
+    height: calc(8*var(--height));
+    width: calc(60*var(--width));
```

Comment:
How does this works?

Reply:
i have set .25rem in variable and based on that i am calculating height and width so it will always a multiplier of .25rem. Keeping it concise.

Reply:
Nice, learned new thing.